README.md

```diff
@@ -1,92 +1,123 @@
+Load data into [CKAN DataStore](https://docs.ckan.org/en/2.8/maintaining/datastore.html) using Airflow as the runner. This is a replacement for DataPusher and Xloader.
 
-Clean separation of components do you can reuse what you want (e.g. don't use Airflow but use your own runner)
+Clean separation of components so you can reuse what you want (e.g., you don't use Airflow but your own runner).
 
 <!-- toc -->
 
-- [Get Started](#get-started)
-- [Examples](#examples)
-  * [Example 1: CSV to JSON](#example-1-csv-to-json)
-  * [Example 2: Local file to CKAN DataStore](#example-2-local-file-to-ckan-datastore)
-    + [Preliminaries: Setup your CKAN instance](#preliminaries-setup-your-ckan-instance)
-    + [Doing the load](#doing-the-load)
-  * [Example 2a: Remote file to DataStore](#example-2a-remote-file-to-datastore)
-  * [Examples 3: Auto Load file uploaded to CKAN into CKAN DataStore](#examples-3-auto-load-file-uploaded-to-ckan-into-ckan-datastore)
-    + [Run it](#run-it)
-- [Tutorials](#tutorials)
-  * [Using Google Cloud Composer](#using-google-cloud-composer)
+- [💨🥫 AirCan](#-aircan)
+  - [Get Started](#get-started)
+  - [Examples](#examples)
+    - [Example 1: CSV to JSON](#example-1-csv-to-json)
+  - [Using Aircan DAGs](#using-aircan-dags)
+    - [Example 2: Local file to CKAN DataStore using the Datastore API](#example-2-local-file-to-ckan-datastore-using-the-datastore-api)
+      - [Preliminaries: Setup your CKAN instance](#preliminaries-setup-your-ckan-instance)
+    - [Example 3: Local file to CKAN DataStore using Postgres](#example-3-local-file-to-ckan-datastore-using-postgres)
+      - [Preliminaries: Setup your CKAN instance](#preliminaries-setup-your-ckan-instance-1)
+      - [Doing the load](#doing-the-load)
+    - [Example 2a: Remote file to DataStore](#example-2a-remote-file-to-datastore)
+    - [Examples 3: Auto Load file uploaded to CKAN into CKAN DataStore](#examples-3-auto-load-file-uploaded-to-ckan-into-ckan-datastore)
+      - [Run it](#run-it)
+  - [Tutorials](#tutorials)
+    - [Using Google Cloud Composer](#using-google-cloud-composer)
 
 <!-- tocstop -->
 
 ## Get Started
 
-* Install Python >= 3.xx (and make a virtualenv)
-* Clone aircan so you have the examples
-  ```
+* Install **Python** >= 3.5 (and make a virtual environment).
+* Clone `aircan` so you have examples available:
+
+  ```bash
   git clone https://github.com/datopian/aircan
   ```
-* Airflow: install and setup https://airflow.apache.org/docs/stable/installation.html
-  ```
+
+* Install and setup **Airflow** (https://airflow.apache.org/docs/stable/installation.html):
+
+  ```bash
   export AIRFLOW_HOME=~/airflow
   pip install apache-airflow
   airflow initdb
   ```
-* Start the server and visit your airflow admin UI
-  ```
+  
+_Note_: On recent versions of Python (3.7+), you may face the following error when executing `airflow initdb`:
```

Comment:
This may also be caused by the fact I tried this with Python 3.8. In either case, this is just a note and it fixes the problem easily. Python 3.8 was actually released back in October 14th, 2019, so more and more people might be running into issues with newer versions.